src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV1.cs

```diff
@@ -44,7 +44,7 @@ protected static class ArtifactEventProperties
             public static readonly string Tags = "tags";
             public static readonly string ArtifactName = "artifactName";
             public static readonly string ItemPattern = "itemPattern";
-            public static readonly string DownloadPath = "downloadPath";
+            public static readonly string TargetPath = "targetPath";
         }
     }
```

Comment:
Does the existing inputs work, or do we need to make changes at https://dev.azure.com/mseng/_git/AzureDevOps?path=%2FReleaseManagement%2FService%2FReleaseManagement2%2FServer%2FDistributedTask%2FArtifactTaskMapper.cs&version=GBmaster&line=377&lineStyle=plain&lineEnd=394&lineStartColumn=1&lineEndColumn=1 too?

Reply:
@omeshp , is this for the Auto Injection? If yes, then we might have to change that as well. But we are adding an alias for the existing input should work. @owenhuynMSFT - FYI. We might want to test all such scenarios.

Reply:
We have modified the agent to work with "downloadPath as well, so you should be good. Eventually it would be better if we change it to targetPath.

Reply:
@fadnavistanmay Yes this is for the auto-inject flow in RM.

Reply:
@fadnavistanmay today's issue is same as what @omeshp pointed in this PR.

Reply:
Hi Lova, right, hence I thought that the alias would work in RM as well.